sriov/tests/common/test_exec.py

```diff
@@ -31,9 +31,11 @@ def test_execute_cmd_fail(dut):
 
 
 def test_execute_cmd_timeout(dut):
-    cmd = "sleep 10s"
+    cmd = "sleep 6s"
     dut.log_str(cmd)
     code, out, err = dut.execute(cmd)
+    # Allow for original sleep to end (5 second timeout + 1 second) before cleanup
+    time.sleep(1)
```

Comment:
is it safer to sleep a little longer than 1? like 1.5?

Reply:
Done in the most recent commit